bindata/etcd/pod.yaml

```
@@ -311,6 +311,31 @@ ${COMPUTED_ENV_VARS}
         name: log-dir
       - mountPath: /etc/kubernetes/static-pod-certs
         name: cert-dir
+  - name: etcd-backup-server
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -euo pipefail
+        
+        exec nice -n -18 cluster-etcd-operator backup-server
```

you can leave it at default niceness, no need to have it fight with grpcproxy or the networking :)
```suggestion
        command: [cluster-etcd-operator]
        args:
        - backup-server
```

---

and remove the rest of the shell script

---

done 👍🏽